src/ManifestStrategy/WireNavigationManifestStrategy.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\ManifestStrategy;
+
+use SimpleXMLElement;
+use SprykerSdk\Integrator\Dependency\Console\InputOutputInterface;
+use SprykerSdk\Integrator\Exception\UnexpectedNavigationXmlStructureException;
+
+class WireNavigationManifestStrategy extends AbstractManifestStrategy
+{
+    /**
+     * @var string
+     */
+    protected const TARGET_NAVIGATION_FILE = 'config/Zed/navigation.xml';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATIONS_CONFIGURATION = 'navigations';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATION_POSITION_AFTER = 'after';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATION_POSITION_BEFORE = 'before';
+
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return 'wire-navigation';
+    }
+
+    /**
+     * @param array<mixed> $manifest
+     * @param string $moduleName
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     * @param bool $isDry
+     *
+     * @return bool
+     */
+    public function apply(array $manifest, string $moduleName, InputOutputInterface $inputOutput, bool $isDry): bool
+    {
+        if (!file_exists($this->getNavigationSourceFilePath())) {
```

Comment:
If this file is not present that means the project uses another way of building navigation. And newly created file can break everything (I don't know how the creating of navigation works). I decided to do in this way. What do you think?

Reply:
I think it can not break anything on the project if we would create this file. When they use another way for the navigation then a new schema file shouldn't be able to break anything.